<ui version="4.0" >
 <class>NMGStatisticRequestWidgetBase</class>
 <widget class="QWidget" name="NMGStatisticRequestWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Path</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPath" />
     </item>
     <item>
      <widget class="QPushButton" name="btnFile" >
       <property name="minimumSize" >
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="btnRequest" >
       <property name="text" >
        <string>Request Statistics</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSubscribe" >
       <property name="text" >
        <string>Subscribe Statistics</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_2_2" >
     <property name="text" >
      <string>Result:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="txtResult" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>txtPath</tabstop>
  <tabstop>btnRequest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
